README.md

```diff
@@ -560,6 +560,48 @@ Partition 1
   none
 ```
 
+## reboot
+
+`reboot` allows you to reboot a device that is in BOOTSEL mode, or in application mode running compatible code (see [Forced Reboots](#forced-reboots)). You can reboot it into application or BOOTSEL mode, and select the Arm or Risc-V CPU on RP2350
+
+```text
+$ picotool help reboot
+REBOOT:
+    Reboot the device
+
+SYNOPSIS:
+    picotool reboot [-a] [-u] [-g <partition>] [-c <cpu>] [device-selection]
+
+OPTIONS:
+    Reboot type
+        -a, --application
+            Reboot back into the application (this is the default)
+        -u, --usb
+            Reboot back into BOOTSEL mode
+        -g <partition>
+            Select diagnostic partition
```

Comment:
@kilograham Could we get a brief explanation of what the diagnostic partition is please?